### ISSUE TITLE
Collections API - Update Multiple Items documentation inaccurate

### ISSUE THREAD

**Issue by the-other-dev** [directus/directus#22982]
### Page

https://docs.directus.io/reference/items.html

### Describe the Inaccuracy

The documentation for [_Update Multiple Items_](https://docs.directus.io/reference/items.html#update-multiple-items) says that the request body should be a partial item object. However, looking at the [code](https://github.com/directus/directus/blob/main/api/src/services/collections.ts#L453-L463) there is an explicit check if the given payload is an array.

The examples for _Update Multiple Items_ look like they got copy/pasted from the singleton example. Should the documentation be updated to reflect the status quo or is the code wrong and needs fixing?

For example the following request works as expected (using an array as the request body):
```
PATCH https://my-directus-instance.example.org/items/foobar
Accept: application/json
Content-Type: application/json
Authorization: Bearer my-secret-token

[
    {
        "id": 1,
        "my_text_field": "value is updated"
    }
]
```

This request however does not work (single item object):
```
PATCH https://my-directus-instance.example.org/items/foobar
Accept: application/json
Content-Type: application/json
Authorization: Bearer my-secret-token

{
    "id": 1,
    "my_text_field": "value is updated"
}
```

and results in:
```json
{
  "errors": [
    {
      "message": "Invalid payload. \"data\" is required.",
      "extensions": {
        "code": "INVALID_PAYLOAD",
        "reason": "\"data\" is required"
      }
    }
  ]
}
```

**Comment by paescuj** [comment]
I agree that the sentence "Provide a partial item object as the body of your request." in the [Request](https://docs.directus.io/reference/items.html#request-7) section is a bit misleading and should be clarified.

However, both the [Request Body](https://docs.directus.io/reference/items.html#request-body-5) and [Example](https://docs.directus.io/reference/items.html#example-7) sections are correctly indicating how the request is to be structured:
```json
{
	"keys": [1, 2],
	"data": {
		"status": "published"
	}
}
```

**Comment by the-other-dev** [comment]
Then it would be great, if both use cases are mentioned in the documentation.
A) Same data for all items (object with key array and same data)
B) Different data for each item (array of partial item objects)

**Comment by rijkvanzanten** [comment]
This is technically a duplicate of https://github.com/directus/directus/issues/22982, but this issue has more details so I'll close the other one instead 🙂

**Comment by phazonoverload** [comment]
This has been fixed in our OpenAPI definition and therefore is fixed in our new docs' API Reference. Thank you!
